src/socket/controller/ingame.ts

```diff
@@ -41,6 +42,10 @@ function handleMove(
     if (data.skill) {
         character.isSkillInput = true
     }
+    if (character.items[0] === ItemType.THUNDER) {
+        gameMap.handleItemUse(character)
+    }
+    character.useItem()
```

Comment:
```suggestion

if(data.item){
    if (character.items[0] === ItemType.THUNDER) {
        gameMap.handleItemUse(character)
    }
    character.useItem()
}
```

* item 사용시에만, 적용되어야할거 같아요!
* 이것도 추후 큐 리스트로 처리되겠죠?

Reply:
헉 완전 놓쳤던부분 땡큐 얘도 큐로 바꿀게

Reply:
아 근데 큐로는 바꿀필요없긴하겠다 우리 순서 충돌나는 아이템 로직이 없어